### PR TITLE
Improve the site's header 

### PR DESCRIPTION
**Pull request by saurabhbakolia**
Improved sites header. 
Search bar, Home, button, profile, button and profile menu. added. 

image

### CODE REVIEW

**Comment by narayan954** [comment]
@saurabhbakolia with this pull request some new issues were introduced, please look at them.
some of them are pointed in my ss
pr review
1. Uneven alignment
2. unnecessary red box
3. input field covers the emoji option
4. Need padding 
5. space left uncovered, needed to fill
6. Uneven alignment

**Comment by saurabhbakolia** [comment]
ok, I am changing it.

**Comment by saurabhbakolia** [comment]
image

image

How it is looking ? Any change you want?

**Comment by narayan954** [comment]
> Fixed all of it. This all happened due of 2 lines of code. 2. unnecessary red box 3. input field covers the emoji option 4. Need padding 5. space left uncovered, needed to fill 6. Uneven alignment
> 
> In 1, I think the alignment should be correct as par the code. But the icons are creating problem. So, I am changing the alignment to flex-start.
> 
> image

yes, you're right, they need css fix. Also, the new post icon is lacking the boldness than others.
And these icons should have a label to be displayed when the user hover on them.
Dummygram logo should also be clickable and work as the home button.

**Comment by saurabhbakolia** [comment]
image

image

Modal design changed as you told. header is sticky now, header icons stroke changed. But I am facing one problem. 
Now when I click on profile icon, as header has a property `overflow:hidden;` and because of that it is preventing profile menu to open.  I need help for that. @narayansoni

**Comment by narayan954** [comment]
> image
> 
> image
> 
> Modal design changed as you told. header is sticky now, header icons stroke changed. But I am facing one problem. Now when I click on profile icon, as header has a property `overflow:hidden;` and because of that it is preventing profile menu to open. I need help for that. @narayansoni

You can show the code/push the code to this pr, so I can try finding the issue. 
If `overflow:hidden;` is the actual cause, then maybe changing it to `overflow:visible;` can help in this case.

**Comment by saurabhbakolia** [comment]
header icons popover text


I used ant design for popover effect.

**Review by narayan954 — CHANGES_REQUESTED** [review]

File: src/App.jsx [hunk 1]
```diff
@@ -9,12 +9,17 @@ import {
   DialogTitle,
   DialogContent,
 } from "@mui/material";
-import { makeStyles } from "@mui/styles";
```
[the remark this answers is not in the document]
Comment (saurabhbakolia) [reply src/App.jsx:12]: $ git pull
fatal: unable to access 'https://github.com/narayan954/dummygram.git/': Failed to connect to github.com port 443 after 18973 ms: Timed out

I don't know why but I am getting this issue. many times.  whenever I try to pull the repo before push. Even I am not able to push the code.

[the remark this answers is not in the document]
Comment (narayan954) [reply src/App.jsx:12]: It's because you haven't set your remote, just google this error, 
tldr - You're trying to directly push to my repo instead of your copy of my repo

[the remark this answers is not in the document]
Comment (narayan954) [reply src/App.jsx:12]: > It might be because you haven't set your remote, just google to set remote properly,

tldr - You're trying to directly push to my repo instead of your copy of my repo

[the remark this answers is not in the document]
Comment (saurabhbakolia) [reply src/App.jsx:12]: but I have added remote origin

[the remark this answers is not in the document]
Comment (saurabhbakolia) [reply src/App.jsx:12]: image

[the remark this answers is not in the document]
Comment (saurabhbakolia) [reply src/App.jsx:12]: image


And How to resolve these.

[the remark this answers is not in the document]
Comment (narayan954) [reply src/App.jsx:12]: that's fake positive don't worry about that

[the remark this answers is not in the document]
Comment (narayan954) [reply src/App.jsx:12]: > but I have added remote origin

yeah, that's not because of git push but git pull

[the remark this answers is not in the document]
Comment (narayan954) [reply src/App.jsx:12]: > Failed to connect to github.com port 443 after 18973 ms: Timed out

try https://stackoverflow.com/questions/18356502/github-failed-to-connect-to-github-443-windows-failed-to-connect-to-github

**Comment by saurabhbakolia** [comment]
image

now working

**Comment by narayan954** [comment]
Were you able to fix the profile button?

**Comment by saurabhbakolia** [comment]
No. And I got some other errors also.

**Comment by narayan954** [comment]
> No. And I got some other errors also.

OH, and I couldn't take a clear look at the issue, some CSS might be conflicting. Try fixing that or something else.
